zulipterminal/cli/run.py

```diff
@@ -285,6 +286,19 @@ def main(options: Optional[List[str]]=None) -> None:
         else:
             theme_to_use = zterm['theme']
 
+        ZULIPRC_CONFIG = 'in zuliprc file'
+        if(zterm['footlinks'][1] == ZULIPRC_CONFIG
+           and zterm['maximum-footlinks'][1] == ZULIPRC_CONFIG):
+            print(in_color(color='red',
+                  text='Footlinks property is not allowed '
+                  'alongside maximum-footlinks'))
+            sys.exit(1)
+        elif(zterm['maximum-footlinks'][1] == ZULIPRC_CONFIG
```

Comment:
This is clearer as just an `if`.

Please add a space between `if`/`elif` and the conditional part.